### PR TITLE
Whirlwind

### PR DESCRIPTION
**Pull request by nick-wilson95**

### CODE REVIEW

**Comment by nick-wilson95** [comment]
[Add whirlwind ability](https://trello.com/c/ca4waSQQ/8-add-whirlwind-ability)

File: scripts/Whirlwind_ContinueChannel/Whirlwind_ContinueChannel.gml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+if(is_undefined(_num_damageCountDown) || _num_damageCountDown == 0) {
+	_num_damageCountDown = WHIRLWIND_DAMAGE_INTERVAL;
+	
+	ds_ContextBag_Set(_ds_contextBag_bag, Whirlwind_ContinueChannel, "damageCountDown", _num_damageCountDown);
+
+	for(i=0; i<ds_list_size(_list_id_collisions); i++) {
+		DamageManager_ApplyDamage(
+			_id_caster,
+			_list_id_collisions[| i],
+			DamageData_New(Enum_DamageTypes.SLASHING, WHIRLWIND_DAMAGE)
```
Comment (pipkin94) [inline scripts/Whirlwind_ContinueChannel/Whirlwind_ContinueChannel.gml]: ds_DamageData_New.
Currently get compile error.

Comment (pipkin94) [inline scripts/Whirlwind_ContinueChannel/Whirlwind_ContinueChannel.gml]: Additionally this doesnt contain all arguments for a DamageData, it doesnt input the snapshot

File: scripts/MACROS_oWhirlwind/MACROS_oWhirlwind.gml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#macro WHIRLWIND_DAMAGE_RADIUS 32
+#macro WHIRLWIND_DAMAGE 15
```
Comment (pipkin94) [inline scripts/MACROS_oWhirlwind/MACROS_oWhirlwind.gml]: Just a personal opinion but it feels very weak haha

**Pull request merged**